src/views/acquirer/legal-gateway.njk

```diff
@@ -0,0 +1,118 @@
+{% extends "page.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{%- from "govuk/components/textarea/macro.njk" import govukTextarea -%}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l">Request ID: {{requestId}}</span>
+        <form method="POST" id="legalGatewayForm">
+        
+        {% set legalGatewayDecision1 %}
+        {{ govukTextarea({
+            id: "yes-decision",
+            name: "yes-decision",
+            type: "text",
+            classes: "govuk-!-width-two-third",
+            label: {
+                text: "What is the legal gateway for acquiring this data?"
+            },
+            value: savedValue['yes-decision']['explanation']
+        }) }}
+        {% endset -%}
+
+        {% set legalGatewayDecision2 %}
+        {{ govukTextarea({
+            id: "other-decision",
+            name: "other-decision",
+            type: "text",
+            classes: "govuk-!-width-two-third",
+            label: {
+                text: "What other legal grounds do you have for acquiring this data?"
+            },
+            value: savedValue['other-decision']['explanation']
+        }) }}
+        {% endset -%}
+
+        {% set legalGatewayDecision3 %}
+        {{ govukTextarea({
+            id: "dont-know-decision",
+            name: "dont-know-decision",
+            type: "text",
+            classes: "govuk-!-width-two-third",
+            value: savedValue['dont-know-decision']['explanation']
+        }) }}
+        {% endset -%}
+
+        {{ govukRadios({
+        name: "legal-gateway",
+        fieldset: {
+            legend: {
+            text: "Is there a 'legal gateway' for acquiring this data?",
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+            }
+        },
+        hint: {
+            text: "Your legal advisor may be asked to support this."
+        },
+        items: [
+            {
+            value: "yes-decision",
+            text: "Yes, there is a legal gateway for acquiring this data",
+            conditional: {
+                html: legalGatewayDecision1
+            }
```

Comment:
Already mentioned this on Slack, but we need to add:
`checked: savedValue['yes-decision']['checked']` to this radio button, and something similar to the others two